### PR TITLE
[docs] Fix context provide decorator api doc

### PR DESCRIPTION
**Pull request by augustjk**

### CODE REVIEW

File: packages/lit-dev-content/site/docs/v2/data/context.md [hunk 1]
```diff
@@ -369,7 +369,7 @@ The `ValueType` type parameter is the type of value that can be provided by this
 
 ### `@provide()`
 
-A property decorator that adds a ContextConsumer controller to the component which will try and retrieve a value for the property via the Context API.
+A property decorator that adds a ContextProvider controller to the component making it respond to any `context-request` events from its children consumer.
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/site/docs/v2/data/context.md:372]: Nit: thoughts about `children consumer` -> `child consumers`?

**Review by AndrewJakubowicz — APPROVED** [review]
My comment is non blocking because this is a pretty glaring error you have fixed. Thank you!

File: packages/lit-dev-content/site/docs/v2/data/context.md (hunk 1 above)
Comment (augustjk) [reply packages/lit-dev-content/site/docs/v2/data/context.md:372]: I took it straight from the JSDoc here
https://github.com/lit/lit/blob/f60a3a2c994f41fc3df1bd8a76451ea185b66e11/packages/context/src/lib/decorators/provide.ts#L19-L20

While I don't oppose to the new wording itself, I think it's minor enough that I'd rather keep it in sync with what's on the source code. Thanks for the suggestion though!

Comment (AndrewJakubowicz) [reply packages/lit-dev-content/site/docs/v2/data/context.md:372]: Great call!

**Pull request merged**